source/help/messaging/sending-messages.rst

```diff
@@ -13,7 +13,7 @@ Write a message by typing into the text input box, then press ENTER to send it.
 Replying to Messages
 ---------------------
 
-Click the **[...]** link next to a message, then click **Edit**. This keeps conversations organized in threads.
+Click the **[...]** link next to a message, then click **Reply**. This keeps conversations organized in threads.
```

Comment:
Thank you also for the review you have done here, these have been really helpful! Will make the changes you requested.